src/main/scala/prci/ClockBundles.scala

```diff
@@ -0,0 +1,16 @@
+// See LICENSE.SiFive for license details.
+package freechips.rocketchip.prci
+
+import chisel3._
+import freechips.rocketchip.util.HeterogeneousBag
+
+class ClockBundle(val params: ClockBundleParameters) extends Bundle
+{
+  val clock = Clock()
+  val reset = Bool()
```

Comment:
Future work. I think we will always pick between Bool/AsyncReset based on a parameter that flows from source to sink, and never rely on `Reset` inference, but that needs to be finalized. It would also break backwards compatibility with the version in fpga-shells, so we might want to reason about whether merging them first and then change it is the right order of operations or not